packages/commerce-sdk-react/README.md

```diff
@@ -112,6 +112,41 @@ _💡 This section assumes you have read and completed the [Authorization for Sh
 
 To help reduce boilerplate code for managing shopper authentication, by default, this library automatically initializes shopper session and manages the tokens for developers. Currently, the library supports the [Public Client login flow](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-public-client.html).
 
+Commerce-react-sdk supports both public and private flow of the [Authorization for Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/authorization-for-shopper-apis.html) guide._
+You can choose to use either public or private slas to login. By default, public flow is enabled.
+
+#### How private SLAS works
+This section assumes you read and understand how private SLAS flow works
```

Comment:
We could link to the SLAS docs here (https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-private-client.html) as an FYI for those who are not familiar with the private flow